kivy/core/window/window_sdl2.py

```diff
@@ -772,18 +775,21 @@ def mainloop(self):
                 Logger.trace('WindowSDL: Unhandled event %s' % str(event))
 
     def _dispatch_drop_event(self, action, args):
+        x, y = (0, 0) if self._drop_pos is None else self._drop_pos
```

Comment:
I think we should bake in to the docs that the default if it's not in the window or if it's not a mouse but e.g. a touch, that it will be `-1` rather than `0`!?

Reply:
Negative values are valid as they can be used to test if the drop occurred on the title bar or on the left window edget. Docs do state that `x` and `y` are the cursor positions at the time of the drop and SDL does not send those values with the `dropfile` event, only `filename` is sent so we don't know from where the drop originated.